### PR TITLE
General refactoring for consistency

### PR DESCRIPTION
**Pull request by ericcornelissen**
Some general refactoring to improve consistency in the JavaScript throughout the project, beyond Prettier. Suggestions are welcome, changes are, obviously, open for discussion

### CODE REVIEW

File: scripts/build-testpage.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
-// core packages
 const fs = require('fs');
-const path = require('path');
-
-// npm packages
 const pug = require('pug');
+const path = require('path');
```
Comment (mondeja) [inline scripts/build-testpage.js:3]: `path` is not a core package?

File: scripts/build.js [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-const util = require('util');
-
-const CleanCSS = require('clean-css');
 const { ucs2 } = require('punycode');
 const SimpleIcons = require('simple-icons');
 const svg2ttf = require('svg2ttf');
 const SVGPath = require('svgpath');
 const ttf2woff = require('ttf2woff');
 const ttf2woff2 = require('ttf2woff2');
+const util = require('util');
```
Comment (mondeja) [inline scripts/build.js:10]: `util` is not a builtin NodeJS package?

**Comment by ericcornelissen** [comment]
@mondeja I dropped the convention of spitting core and NPM packages as I don't find it particularly useful and is hard to enforce, I'd be happy to revert it though as the `require`s shouldn't change too often anyway.

**Review by mondeja — APPROVED** [review]
> I dropped the convention of spitting core and NPM packages as I don't find it particularly useful and is hard to enforce, I'd be happy to revert it though as the requires shouldn't change too often anyway.

Okay, since are scripts I suppose that is not necessary so much order in this. Thanks for this refactorization @ericcornelissen, feel free to merge it if is completed :+1:

**Pull request merged**